src/flp/types/fixedpoint_l2.rs

```diff
@@ -153,16 +153,30 @@
 //! For example, it is `0.999969482421875` for `n = 16`.
 
 pub mod compatible_float;
+pub mod noise;
 
 use crate::field::{FftFriendlyFieldElement, FieldElementExt, FieldElementWithInteger};
 use crate::flp::gadgets::{BlindPolyEval, ParallelSumGadget, PolyEval};
 use crate::flp::types::fixedpoint_l2::compatible_float::CompatibleFloat;
+use crate::flp::types::fixedpoint_l2::noise::sample_discrete_gaussian;
 use crate::flp::{FlpError, Gadget, Type};
 use crate::polynomial::poly_range_check;
 use fixed::traits::Fixed;
+use num_bigint::{BigInt, BigUint, TryFromBigIntError};
 
 use std::{convert::TryFrom, convert::TryInto, fmt::Debug, marker::PhantomData};
 
+use self::noise::compute_std_deviation;
+
+/// The privacy parameter which is passed to `FixedPointBoundedL2VecSum` has this type.
+pub type PrivacyParameterType = (u128, u128);
+
+/// If no noise should be added during aggregation, use the value returned by this function as
+/// privacy parameter.
+pub fn zero_privacy_parameter() -> PrivacyParameterType {
```

Comment:
This ought to be a method on `PrivacyParameterType`, e.g.:

```rust
impl PrivacyParameterType {
   pub(crate) fn no_noise() -> Self { 
      blah!()
   }
}
```

Reply:
Making this the `Default` implementation is a little scary, since depending on your definition of privacy, this may not be a secure default.